docs/developer/binarywheels.rst

```diff
@@ -0,0 +1,94 @@
+Building binary wheels
+======================
+
+Several packages in the Zope Foundation repositories contain C code, which
+means they should be packaged and published as binary wheels alongside the
+standard egg archive. Building binary wheels has its pitfalls, though. If not
```

Comment:
"standard egg archive" sounds a bit weird, given that setuptools `.egg` files have been deprecated for like a decade now.

We tend to publish sdists and (universal) wheels for packages that don't contain C code.  I'm not sure how to call them, to contrast with the multiple platform-specific binary wheels we need for the latter.  Architecture-independent wheels?

Reply:
Or maybe "source distributions"?

Reply:
A source distribution, or sdist, is a very loosely standardized archive (.tar.gz *is* now standardized) containing a `setup.py` file that is executed and can do literally anything. OK, now it's actually more complicated than that, what with the existence of 'pyproject.toml' and PEP 517, but the idea is the same: to install a source distribution, you have to execute arbitrary code.

A wheel, in contrast, is highly standardized and doesn't require executing arbitrary code to install; it contains enough metadata that the installation tool can do it all. It may contain native extensions, or just .py files, or a mix of both. 

The wheel spec refers to the part of the filename that identifies where the wheel can be installed as the "platform tag", e.g., "manylinux" or, for a pure-python package, "any" (I'm ignoring the "ABI tag"). So perhaps we can say "platform-independent wheels"?

Reply:
I removed the half sentence to avoid confusion.